tvtropesTableCreator.py

```diff
@@ -0,0 +1,58 @@
+#py
+# -*- coding: utf-8 -*-
+import requests
+from bs4 import BeautifulSoup
+import sys
+
+def createFile(title, contents):
```

Comment:
PEP8: Function names should be written following the conventon `lower_case_with_underscores`